.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Build RTSan Libs
+
+on:
+  push:
+    tags: ["v*"]
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
I think `ubuntu-latest` is x86 machine? We need arm too I believe.

Reply:
Thanks! I did a quick check on Saturday and this works for us. I'm going to keep it until after this review is merged before adding it in.